src/js/tabs/failsafe.js

```diff
@@ -304,6 +304,10 @@ failsafe.initialize = function (callback) {
             $('input[name="gps_rescue_min_dth"]').attr("min", 20);
         }
 
+        if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_46)) {
+            $('input[name="gps_rescue_min_dth"]').attr("min", 10);
+        }
+
```

Comment:
I think is cleaner to "join" in some way with the earlier if, because both affect the same paremeter. Maybe a simply `else` or a binary conditional operator or similar is enough.

Reply:
Fixed :+1: